homeassistant/components/litejet/config_flow.py

```diff
@@ -62,6 +78,20 @@ async def async_step_user(
             try:
                 system = await pylitejet.open(port)
             except SerialException:
+                if self.context["source"] == config_entries.SOURCE_IMPORT:
+                    async_create_issue(
+                        self.hass,
+                        DOMAIN,
+                        "deprecated_yaml_serial_exception",
+                        breaks_in_ha_version="2024.2.0",
+                        is_fixable=False,
+                        issue_domain=DOMAIN,
+                        severity=IssueSeverity.ERROR,
+                        translation_key="deprecated_yaml_serial_exception",
+                        translation_placeholders={
+                            "set_up_the_integration": "[set up the integration](/config/integrations/dashboard/add?domain=litejet)"
```

Comment:
[At the risk of massively overindexing on making this edge case of a low usage integration be completely shiny, one more comment. Worth it at least so no one randomly copies a bad pattern into somewhere that it really matters.]

This now has the opposite problem: the "set up the integration" text can't be translated since it is coming from the placeholder. The way to do this is to provide only the URL part of the markdown link via the placeholder. See, for example, how [helpers](https://github.com/home-assistant/core/tree/790523126effe84de725b469f3dd3946709719ca/homeassistant/helpers) does URL placeholders.

Reply:
Right. Not sure what I was thinking. Fixed now